src/shared/tpm2-util.h

```diff
@@ -86,8 +90,10 @@ struct tpm2_context;
 static inline int tpm2_not_supported(void) {
         return log_error_errno(SYNTHETIC_ERRNO(EOPNOTSUPP), "TPM2 not supported on this build.");
 }
-#define tpm2_context_init(...) tpm2_not_supported()
-#define tpm2_context_destroy(...) tpm2_not_supported()
+
+#define tpm2_context_alloc(...) tpm2_not_supported()
+#define tpm2_context_free(...) ({ tpm2_not_supported(); NULL; })
```

Comment:
as above, please use static inline for all such stubs to retain type-safety and execute side-effects on parameters

Reply:
ack